src/simulators/statevector/qv_avx2.cpp

```diff
@@ -11,13 +11,15 @@
  * copyright notice, and modified files need to carry a notice indicating
  * that they have been altered from the originals.
  */
-
+#include <iostream>
 #include "qv_avx2.hpp"
 #include <immintrin.h>
 #include <cstdint>
 #include <cstring>
 #include <type_traits>
 #include <utility>
+#include <complex>
+#include <omp.h>
```

Comment:
I think this needs a an `#ifdef` guard:
```suggestion
#ifdef _OPENMP
#include <omp.h>
#endif
```
same for any part of the code that uses something from this header file